app/Http/Controllers/StatementController.php

```diff
@@ -156,16 +158,16 @@ public function getStatementHistory(Request $request, Validate $validate)
         $response->ifSupportDelayed = null;
         try {
             $response->topic = Camp::getAgreementTopic($filter);
-            $response->topic->go_live_time=date('m/d/Y, h:i:s A', $response->topic->go_live_time);
-            $response->topic->submit_time=date('m/d/Y, h:i:s A', $response->topic->submit_time);
+            $response->topic->go_live_time = date('m/d/Y, h:i:s A', $response->topic->go_live_time);
+            $response->topic->submit_time = date('m/d/Y, h:i:s A', $response->topic->submit_time);
             $response->liveCamp = Camp::getLiveCamp($filter);
-            $response->liveCamp->go_live_time=date('m/d/Y, h:i:s A', $response->liveCamp->go_live_time);
-            $response->liveCamp->submit_time=date('m/d/Y, h:i:s A', $response->liveCamp->submit_time);
+            $response->liveCamp->go_live_time = date('m/d/Y, h:i:s A', $response->liveCamp->go_live_time);
```

Comment:
This is repeated at several places. It is better to create a function. something similar to this (May be in Util)

```php
public function convertDateToXX($unix_time) {
     return date('m/d/Y, h:i:s A', $unix_time);
}
```